HomeWork/src/main/java/generics/Orange.java

```diff
@@ -0,0 +1,21 @@
+package generics;
+
+import java.util.List;
+
+public class Orange extends Fruit {
+
+    private float weightOrange = 1.5F;
```

Comment:
This class has the same mistakes as class Apple. Check the comments and apply changes here.